odom_mode: 0
freq: 20

start_now: 32
start_togo: 13

oriParam: 0
tolerence: 30
decelerationZone: 30
nodeLoseConpDELAY: 15
X_after_over_hurdles: 300
after_6_shift: 40


# constant
span: 0.1
# phy_maxMS: 30
phy_maxMS: 200
slowMS: 15

# soft
softRate: 0.7

# tracker param
black: 0
nonblack: 1

# deceleration
deceleration_coeff: 0.8
deceleration_minSpeed: 5

# variable
# cmd_vel
ori: -1

# V: 30
V: 100
vel_limit: 20
w_limit: 10000

# transmission fix

vkp: 5
vki: 0.0
vkd: 0.0

# rotation fix

wkp: 1.6
wki: 0.005
wkd: 0.0
